showcase/app/templates/components/copy/button.hbs

```diff
@@ -167,7 +167,7 @@
 
   <Shw::Text::H4>Target types</Shw::Text::H4>
 
-  <Shw::Grid @columns={{3}} @gap="2rem" as |SF|>
+  <Shw::Flex @gap="2rem" as |SF|>
```

Comment:
Note: columns isnt an argument on Flex (its only on Grid), opted to change it back to Flex and remove the columns argument to avoid changes in Percy.